packages/wallet/api/src/marshal-contexts.js

```diff
@@ -295,6 +294,9 @@ export const makeImportContext = (makePresence = defaultMakePresence) => {
      * @param {string} iface
      */
     fromMyWallet: (slot, iface) => {
+      if (!slot) {
+        return makePresence('dummy');
```

Comment:
Is this an expected case, or should we assert here or something?

Reply:
Expected case when the other marshaller returns `null` slots like `E(board).getReadonlyMarshaller()`